src/devtools/utils.ts

```diff
@@ -123,6 +123,19 @@ export function useSafeState<T>(initialState: T): [T, (value: T) => void] {
   return [state, safeSetState]
 }
 
+/**
+ * Displays a string regardless the type of the data
+ * @param {any} value Value to be stringified
+ */
+ export const displayValue = (value: any) => {
```

Comment:
let's please use `unknown` instead of `any`:

```suggestion
/**
 * Displays a string regardless the type of the data
 * @param {any} value Value to be stringified
 */
 export const displayValue = (value: unknown) => {
```